lib/readyset/relation_extension.rb

```diff
@@ -10,21 +10,11 @@ module RelationExtension
       # the queries issues to do the eager loading will not have caches created. Those queries must
       # have their caches created separately.
       #
+      # @param [Boolean] whether the queries to this cache should always be served by ReadySet,
+      #                  preventing fallback to the uptream database
       # @return [void]
-      def create_readyset_cache!
-        Readyset.create_cache!(sql: to_sql)
-      end
-
-      # Drops the cache on ReadySet associated with this query. This method is a no-op if a cache
-      # for the query already doesn't exist.
-      #
-      # NOTE: If the ActiveRecord query eager loads associations (e.g. via `#includes`), the
-      # the queries issues to do the eager loading will not have caches dropped. Those queries must
-      # have their caches dropped separately.
-      #
-      # @return [void]
-      def drop_readyset_cache!
```

Comment:
ReadySet is not going to support `DROP CACHE <query text>`, so this method shouldn't exist anymore